test/lib/mayaUsd/fileio/testImportWithNamespace.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env mayapy
```

Comment:
Test Driven Development: without the fix the test fails, after adding the fix the test passes.